### PR TITLE
added base code for ingestor and climber

### PR DESCRIPTION
**Pull request by hjicarus5602**
this commit contains the initial classes for climber and ingestor (will be developed more in days to come)

### CODE REVIEW

**Comment by PNovak619** [comment]
Hi Harish - the climber code seems to be for shooter, or at least references shooter motor.  It should reference the rung 1 winch motor from the Electrical DT file in the Integration Drive.  Let me know if I misread something.

File: src/main/java/frc/robot/Climber.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package frc.robot;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import edu.wpi.first.wpilibj.XboxController;
+
+public class Climber {
+    private WPI_TalonSRX liftGate;
```
Comment (programmer94) [inline src/main/java/frc/robot/Climber.java:8]: Are you sure these variable names are correct?

**Review by programmer94 — CHANGES_REQUESTED** [review]
Confirm or correct these variable names, and I'll approve your PR.

**Review by programmer94 — APPROVED** [review]
Looks fine to me.

**Pull request merged**